src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form_group">
+        <legend>Personal information</legend>
+        <div class="form_content">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
            required
          >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
fix code style all over the page